default.nix

```diff
@@ -195,7 +198,7 @@ in {
       };
 
   in
-    assert pnpmlock.lockfileVersion == 5;
+    assert (pnpmlock.lockfileVersion == 5 || pnpmlock.lockfileVersion == 5.1);
```

Comment:
Just a minor nit (also for myself): We should only check the major version and assume minor will stay compatible.